### PR TITLE
Convert SAP QAM tests to YAML scheduling

### PR DESCRIPTION
**Pull request by ldevulder**
Convert the SLES4SAP QAM tests to use full YAML.

- Related ticket: https://jira.suse.com/browse/TEAM-3955
- Job Groups scheduler: https://gitlab.suse.de/qa-css/openqa_ha_sap/-/merge_requests/291
- Needles: N/A
- Verification run:
[Maintenance: SLE 12 SP4 SAP Incidents](http://1b210.qa.suse.de/tests/overview?build=MR:240001:sapnwbootstrap-formula&distri=sle&version=12-SP4&groupid=22)
[Maintenance: SLE 12 SP5 SAP Incidents](http://1b210.qa.suse.de/tests/overview?distri=sle&version=12-SP5&build=MR%3A240001%3Asapnwbootstrap-formula&groupid=21)
[Maintenance: SLE 15 GA SAP Incidents](http://1b210.qa.suse.de/tests/overview?version=15&build=MR%3A240432%3Asapstartsrv-resource-agents&distri=sle&groupid=20)
[Maintenance: SLE 15 SP1 SAP Incidents](http://1b210.qa.suse.de/tests/overview?distri=sle&groupid=23&build=MR%3A240432%3Asapstartsrv-resource-agents&version=15-SP1)
[Maintenance: SLE 15 SP2 SAP Incidents](http://1b210.qa.suse.de/tests/overview?version=15-SP2&build=MR%3A240432%3Asapstartsrv-resource-agents&groupid=24&distri=sle)

### CODE REVIEW

File: schedule/sles4sap/qam/common/ha_supportserver.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: ha_supportserver
```
Comment (juadk) [inline schedule/sles4sap/qam/common/ha_supportserver.yaml:2]: In all the HA QAM tests, I've prefixed both files and names with `qam_` like:
`name:           qam_ha_qdevice_node` and `schedule/ha/qam/common/qam_ha_qdevice_node.yaml` 
My solution is perhaps not the better but we should agree on a common naming convention.

Comment (ldevulder) [reply schedule/sles4sap/qam/common/ha_supportserver.yaml:2]: Ok, I will prefix all of them.

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Review by juadk — APPROVED** [review]
LGTM

**Pull request merged**